### ISSUE TITLE
Sloped 3D Floors not displaying correctly

### ISSUE THREAD

**Issue by gamegenten** [coelckers/gzdoom#182]
Tested in GZDoom 2.2.0, 2.3.0, and the latest GZDoom-GPL. 
Tested as working correctly in GZDoom 2.1.1

Here they are in 2.1.1 
http://imgur.com/a/OvtDZ
and here they are in GZGPL 1.0. They look the same as this in GZDoom 2.2.0 and 2.3.0  http://imgur.com/a/x7UWD

The top and bottom of the sloped 3d floor are invisible, and the floor and ceiling of said sector appear stretched. Also, under the bridge, I lowered the sector lighting and in these versions it is only darker when I stand in said sector.

Hope this is an okay format, this is my first time reporting a bug here, and I'm unsure if there isn't just some rendering setting that is messing it up.

**Comment by nashmuhandes** [comment]
Please provide a runnable WAD file.

As a general rule of thumb, images are next to useless for debugging purposes in (G)ZDoom.

If you don't want to share the map publically, you can PM Graf Zahl on the forum for the map. Don't forget to reference this issue in your PM.

**Comment by coelckers** [comment]
The screenshot doesn't tell me what this map is doing that is wrong. Be aware that this isn't a general problem, I have played countless maps recently which looked fine.

So yes, I absolutely need the map to see what is wrong here.

**Comment by coelckers** [comment]
Also important: What is your graphics card? There are two processing paths for sloped walls, depending on hardware features.

**Comment by gamegenten** [comment]
K, here is my map WAD, and it is MAP02. 
https://www.dropbox.com/s/2qdas50pyj6dzj9/map02issuewithsloped3dfloors.wad?dl=0

I've moved the PlayerStart to right under the bridge for convenience. 
I didn't batch in my resource file/decorates, so the monsters are conveniently absent. You'll see some carefully placed <!> sprites in the area. If this pose any issue I can then zip it up with my resources and send privately to Graf.
I run this over Freedoom 2 and FreeDM, but I checked it in Doom2 as well, to make sure it looks about the same.
I'm not sure what I could be doing wrong that would still let it work in an older version, other than possibly some random default display setting I need to change?
I have an EVGA geForce GTX 960 SSC.
